content/en/internal/configuration-database/howto-faq/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "HOW TOs and FAQs"
+weight: 30
+description: >-
+     HOW TOs and FAQs.
+---
+
+## I get an "error 12227" message when accessing GOC portal with Mozilla/Firefox
+
+This happens when no certificate has been uploaded to your browser. Refer to the
+"Access to GOCDB" section for more information about GOCDB and X509 certificates.
+
+<!-- markdownlint-disable no-inline-html -->
+## I am responsible for a site that has recently entered the EGI infrastructure. How do I register it?
+<!-- markdownlint-enable no-inline-html -->
+
+Only registered users with an approved role on an NGI can add a new site. If you
+are the site administrator, the first thing to do is to contact your NGI staff
+and ask them to add the site for you. Then, register to GOCDB (see the user
+account section) and ask for a site admin role for your site (see the requesting
+a role section). Once your role approved, you will be able to edit and change
+your site information.
+
+## Why can't I declare downtimes for my whole site as I used to do in GOCDB3?
+
+For data clarity reasons, it has been decided long ago to only link downtimes to
+services, thus avoiding the complication of having to check both site and
+service downtimes to determine whether a service is up or not. The way to
+declare a downtime for your site is to select all the services of the site in
+one go when inserting the downtime.
+
+## How do I extend a declared schedule downtime?
+
+Because of EGI policies it is not possible to extend a downtime. Recommended
+good practice for any downtime extension is to declare a new unscheduled
+downtime, starting just when the frst one finishes. please refere to the
+downtimes section of this documentation for more information, especially the
+"downtime extension" paragraph.
+
+<!-- markdownlint-disable no-inline-html -->
+## I have declared a downtime "at risk", and it turns out to be an outage. How can I declare this properly?
+<!-- markdownlint-enable no-inline-html -->
+
+If you have declared the downtime as being at risk and an outage actually happens
+half way through, you need to update GOCDB to reflect the fact that your site is
+now down. There is currently no way of doing this by updating the downtime on the
+fly without having the system considering the whole downtime as being an outage.
+The best way to proceed is:
+
+## Modify end date of your "at risk" downtime, so that it ends in a few minutes
+
+Enter a new "outage" downtime, starting when the other ends
+How do I switch monitoring on/off for my nodes?
```

Comment:
```suggestion
The best way to proceed is:
- Modify end date of your "at risk" downtime, so that it ends in a few minutes
- Enter a new "outage" downtime, starting when the other ends

## How do I switch monitoring on/off for my nodes?
```

Think the markdown here got slightly garbled.